File: TestRestApi/src/main/java/com/amuyu/testrestapi/TestRestApi.java
package com.amuyu.testrestapi;

import android.content.Context;

import com.amuyu.logger.DefaultLogPrinter;
import com.amuyu.logger.Logger;


/**
 * Created by noco on 2016-10-12.
 */
public class TestRestApi {

    public static void init(Context context) {
        Logger.addLogPrinter(new DefaultLogPrinter(context));
    }
}

File: TestRestApi/src/main/java/com/amuyu/testrestapi/TestRestApi.kt
package com.amuyu.testrestapi

import android.content.Context
import com.amuyu.logger.DefaultLogPrinter
import com.amuyu.logger.Logger


/**
 * Created by noco on 2016-10-12.
 */
object TestRestApi {

    fun init(context: Context) {
        Logger.addLogPrinter(DefaultLogPrinter(context))
    }

    fun test(): Int? {
        var temp: String? = null
//        val size = temp?.length ?: -1
        val size = temp?.length
        return size
    }
}
